src/ctia/stores/es/crud.clj

```diff
@@ -457,15 +457,20 @@ It returns the documents with full hits meta data including the real index in wh
 (s/defn refine-full-text-query-parts :- [{s/Keyword ESQFullTextQuery}]
   [full-text-terms :- [FullTextQuery]
    default-operator]
-  (let [term->es-query-part (fn [{:keys [query_mode] :as x}]
+  (let [term->es-query-part (fn [{:keys [simple_query
```

Comment:
🔴  You have a bug here:
this fn receives a FullTextQuery as input:
``` clojure
(s/defschema FullTextQuery
  (st/merge
   {:query s/Str}
   (st/optional-keys
    {:query_mode       FullTextQueryMode
     :fields           [(s/enum s/Keyword s/Str)]
     :default_operator s/Str})))
```

Reply:
Aye. Good catch. Indeed, it was very bad of me to miss that. My sincere apologies. I've fixed it and added a simple test to `ctia.stores.es.crud-test` for the function.